tracing/trace.go

```diff
@@ -64,6 +65,9 @@ func (t *trace) addCounter(key string, delta float64) {
 }
 
 func (t *trace) setTag(key string, value interface{}) {
+	if v, ok := value.(bool); ok {
+		value = interface{}(strconv.FormatBool(v))
```

Comment:
I don't think `interface{}()` is necessary? `value = strconv.FormatBool(v)` should just work?

Reply:
You're right, I thought it would complain about setting an `interface{}` to another value.